src/hackerspace_online/management/commands/lowercase_usernames.py

```diff
@@ -0,0 +1,33 @@
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from django.db import transaction
+
+from django.db import models
+from django.db.models.functions import Lower
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+
+    help = 'Used to lowercase usernames'
```

Comment:
Can we make this loop through all tenants?  Otherwise we have to run in manually for all tenants, there are too many tenants in production...

Reply:
Okay. I thought we are already using `django-tenants` and they provide a custom command

https://django-tenants.readthedocs.io/en/latest/use.html#all-tenants-command

So we could just do

```
python src/manage.py all_tenands_command lowercase_usernames
```

Reply:
Forgot that we still use django-tenant-schemas

Reply:
You are right, we ARE using django-tenants now!  I didn't know about `all_tenands_command` !!